clients/lsp-astro.el

```diff
@@ -0,0 +1,45 @@
+;;; lsp-astro.el --- lsp-mode astro integration -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2022 Paweł Kobojek
+
+;; Author: Paweł Kobojek
+;; Keywords: languages,astro
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;;  client for astro-ls
+
+;;; Code:
+(require 'lsp-mode)
+
+(defgroup lsp-astro nil
+  "LSP support for Astro.build, using astro-ls."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/withastro/language-tools"))
+
+(add-to-list 'lsp-language-id-configuration
```

Comment:
can you move this in lsp-language-id-configuration initialization?

Reply:
Done